cpp/src/slice2swift/Gen.cpp

```diff
@@ -557,7 +554,7 @@ Gen::TypesVisitor::visitStructStart(const StructPtr& p)
 
     out << nl << "/// Write a `" << name << "` structured value to the stream.";
     out << nl << "///";
-    out << nl << "/// - parameter _: `" << name << "` - The value to write to the stream.";
+    out << nl << "/// - Parameter v: The value to write to the stream.";
```

Comment:
Always document the _parameter_ label, not the _argument_ label.
That's backwards of what you'd expect IMO, but that's how it is.
I guess it's avoid awkward situations like this where there is no argument label?